src/pages/LoginPage/index.js

```diff
@@ -0,0 +1,86 @@
+import axios from "axios";
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuth } from "../../contexts/authContext";
+import "./styles.css";
+
+const LoginPage = () => {
+  const { setAuth } = useAuth();
+  const [inputValues, setInputValues] = useState({
+    email: "adarshbalika@gmail.com",
+    password: "adarshbalika",
+  });
+  const navigate = useNavigate();
+  const [error, setError] = useState("");
+
+  const signinHandler = async () => {
+    try {
+      setAuth({
+        isAuthenticated: false,
+        user: "",
+        token: "",
+      });
+      setError("");
+      const response = await axios.post(`/api/auth/login`, {
+        email: inputValues.email,
+        password: inputValues.password,
+      });
+      localStorage.setItem("AUTH_TOKEN", response.data.encodedToken);
+      localStorage.setItem("WB_USER", JSON.stringify(response.data.foundUser));
+      setAuth(a => ({
+        ...a,
+        isAuthenticated: true,
+        user: response.data.foundUser,
+        token: response.data.encodedToken,
+      }));
+      navigate(`/`);
```

Comment:
Just a suggestion, you can add an empty line after lines 22, 27, 29, 35. This makes it cleaner and easier to identify where the code block ends. (according to my opinion)

Reply:
Sure, will do that.